gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,7 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.1-all.zip
+#list: https://services.gradle.org/distributions/
+#curl -L "https://services.gradle.org/distributions/gradle-7.6.2-all.zip.sha256"
+distributionSha256Sum=5861dcc093556fef71cdde8cbae9887be70a8bf40e0f5856b61181ae92dccf33
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.2-all.zip
```

Comment:
I tend to upgrade the wrapper via the:
```
./gradlew wrapper --gradle-version 7.6.2
```
as it also updates the binary which downloads gradle (`gradle-wrapper.jar`), as well as the executables used to run it (`gradlew`)